dd-trace-api/src/main/java/signalfx/trace/api/Trace.java

```diff
@@ -0,0 +1,20 @@
+package signalfx.trace.api;
```

Comment:
Mind using `signalfx.tracing.api` for uniformity?

Reply:
Uniformity with what?  Do we have some other Java code out there with that package name?

Reply:
consistency across instrumentation libraries (signalfx-tracing)

Reply:
Any particular reason why this isn't under `com.signalfx.public.tracing.api` ?

Reply:
We can do `com.signalfx.tracing.api` but the word `public` is not allowed in package names since it is a keyword in Java and each dot-separated component of a package name has to be a valid identifier: https://docs.oracle.com/javase/specs/jls/se8/html/jls-7.html#jls-7.4.1

Reply:
Ah yeah sorry the `public` is in our artifactId. Let's do `com.signalfx.tracing.api` then, if possible.

Reply:
ok, updated to that package name.